install/update_2.6_2.7.php

```diff
@@ -32,6 +32,19 @@
 function plugin_formcreator_update_2_7(Migration $migration) {
    global $DB;
 
+   // Upgrade plugin configuration table
+   $table = 'glpi_plugin_formcreator_entityconfigs';
+   $migration->displayMessage("Upgrade $table");
+   $migration->addField($table, 'external_links_prefix', 'string', ['after' => 'id']);
+   $migration->addField($table, 'external_links_icon', 'string', ['after' => 'id']);
+   $migration->addField($table, 'external_links_title', 'string', ['after' => 'id']);
+   $migration->addField($table, 'tickets_summary', 'integer', ['after' => 'id', 'value' => '1']);
+   $migration->addField($table, 'user_preferences', 'integer', ['after' => 'id', 'value' => '1']);
+   $migration->addField($table, 'avatar', 'integer', ['after' => 'id', 'value' => '1']);
+   $migration->addField($table, 'user_name', 'integer', ['after' => 'id', 'value' => '0']);
+   $migration->addField($table, 'profile_selector', 'integer', ['after' => 'id', 'value' => '1']);
+   $migration->migrationOneTable($table);
```

Comment:
If you don't do any processing in the upgrade 2.6 -> 2.7 requiring that the table is up to date you can remove this line. the instance of Migration will do it for you

Reply:
Ok